aksel.nav.no/website/.storybook/main.ts

```diff
@@ -35,23 +35,42 @@ const config: StorybookConfig = {
       return loadCsf(code, { ...opts, fileName }).parse().indexInputs;
     };
 
+    const templateIndexer = async (fileName: string, opts) => {
+      let code = readFileSync(fileName, "utf-8").toString();
+
+      code = code.replace(/export default .*/g, "");
+
+      code = code.replace("export const args =", "const args =");
+
+      code += `\nexport default { title: "Templates/${fileName
+        .split("pages/templates/")[1]
+        .replace(".tsx", "")}"  };\n`;
+
+      return loadCsf(code, { ...opts, fileName }).parse().indexInputs;
+    };
+
     return [
       ...(indexers || []),
       {
         test: /(stories|story)\.[tj]sx?$/,
         createIndex: csfIndexer,
       },
       {
-        test: /pages\/eksempler\/|(".+")|(.+.tsx)$/,
```

Comment:
tolket denne som feil (way to permissive? med `|(".+")` )

Reply:
Så lenge ingen eksempler forsvinner når du tar det vekk, så tenker jeg at det bør fungere.
Vil du unngå å endre, så kunne sikkert ny matcher vært plassert før eksempler-matcheren, men jeg mener det er bedre med en konkret regex.